src/mame/midway/atlantis.cpp

```diff
@@ -357,8 +355,7 @@ uint8_t atlantis_state::cmos_r(offs_t offset)
 	// Initial RTC check expects reads to the RTC to take some time
 	if (offset == 0x7ff9)
 		m_maincpu->eat_cycles(30);
-	if (LOG_RTC || ((offset >= 0x7ff0) && (offset != 0x7ff9)))
-		logerror("%s:RTC read from offset %04X = %08X\n", machine().describe_context(), offset, result);
+	LOGMASKED(LOG_RTC, "%s:RTC read from offset %04X = %08X\n", machine().describe_context(), offset, result);
```

Comment:
This was previously logging on unexpected offsets as well as when `LOG_RTC` was set.

Reply:
Good catch, I re-added the unexpected-offset check.